tests/inference/qnn_test.py

```diff
@@ -425,12 +428,20 @@ def test_expectation_bitstring_energy(self):
         n_moments=n_moments,
         p=act_fraction)
     state_raw_circuit = state_raw_circuits[0]
-    state_circuit = models.DirectQuantumCircuit(state_raw_circuit)
+    qnn_minval = 0.25
+    qnn_maxval = 0.75
+    state_circuit_initializer = tf.keras.initializers.RandomUniform(
+        minval=qnn_minval, maxval=qnn_maxval, seed=self.tf_random_seed)
+    state_circuit = models.DirectQuantumCircuit(state_raw_circuit,
+                                                state_circuit_initializer)
+    state_circuit.build([])
```

Comment:
jfme: why do we have to call build here now?

Reply:
Removed, tested it and turns out we don't need this one